packaging/fedora/snapd.spec

```diff
@@ -542,7 +542,7 @@ export GO111MODULE=off
 %endif
 
 # Generate version files
-./mkversion.sh "%{version}-%{release}"
+DPKG_PARSECHANGELOG="" ./mkversion.sh "%{version}-%{release}"
```

Comment:
could you instead add `BuildConflicts: alien` ?

Reply:
I think that's just backwards. The problem is that we assume this is a debian system just because it ships a Debian tool. I'd have to remove alien to use it.

Reply:
then perhaps mkversion should use a better check, maybe look for apt? or simply check os-release/call lsb_release

Reply:
apt can be present too

I think this is fine, we don't want to use debian changelog in fedora builds.